### PR TITLE
[Merge request]  bug fix on table structure metric

### PR DESCRIPTION
**Pull request by tbs17**
**Summary**
This fix is to provide better logic oon matched_idx of calculating table structure metric to provide more accurate calculation on the acc
**Additional Context**

- this fix has passed CI run in Draft PR #3025 initially
- therefore, this time we would like to merge into main branch
- this commit has merged the latest change from main after the Draft PR

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.14.5-dev1
+### Enhancements
+### Features
+### Fixes
```
Comment (scanny) [inline CHANGELOG.md:4]: There should be a blank line between each of these, match how it is in the rest of the document, title, blank line, Enhancements, blank line, etc.

File: unstructured/metrics/evaluate.py [hunk 2]
```diff
@@ -161,6 +161,8 @@ def _try_process_document(self, doc: Path) -> Optional[list]:
     def _process_document(self, doc: Path) -> list:
         """Should return all metadata and metrics for a single document."""
 
+        pass
```
Comment (scanny) [inline unstructured/metrics/evaluate.py:164]: No blank line between function and first statement.

File: unstructured/metrics/table/table_alignment.py [hunk 3]
```diff
@@ -50,11 +50,10 @@ def get_table_level_alignment(
 
     @staticmethod
     def _zip_to_dataframe(table_data: List[Dict[str, Any]]) -> pd.DataFrame:
-        df = pd.DataFrame(table_data).pivot(
-            index="row_index",
-            columns="col_index",
-            values="content",
-        )
+
```
Comment (scanny) [inline unstructured/metrics/table/table_alignment.py:53]: same

**Review by scanny — APPROVED** [review]
A few nits on formatting, otherwise LGTM :)

Not sure why ingest tests are failing, try rerunning those, sometimes that works:
- Click on "Details" next to failed job
- Go to top look for "Re-run jobs"
- Select "Re-run failed jobs"

**Comment by MthwRobinson** [comment]
@scanny @tbs17 - Are you still working this PR?

**Comment by tbs17** [comment]
> @scanny @tbs17 - Are you still working this PR?

i think this should be good to go!

**Comment by tbs17** [comment]
merge table metric bug fix

**Pull request merged**